config/locales/custom/budgets.yml

```diff
@@ -40,12 +46,19 @@ en:
       map: The geographical location of budget proposals
     investments:
       form:
-        map_location_instructions: "Tell us where your project idea would be implemented."
+        tags_help_text: ""
+        estimated_price_help_text: ""
+        description_help_text: "Specify the project SMART (Specific, Measurable, Achievable, Relevant, Time-bound) objectives, the target audience it addresses, qualitative and quantitative description, the impact you expect the project to have (how many people and how they will benefit from the project) and the estimated budget. If you have a detailed budget, please upload it in the Documents section."
```

Comment:
"Specify the project objectives in the SMART format (Specific, Measurable, Achievable, Relevant, Time-bound), the target audience it addresses, qualitative and quantitative description, the impact you expect the project to have (how many people and how they will benefit from the project) and the estimated budget. If you have a detailed budget, please upload it in the Documents section."